keep-alive/README.md

```diff
@@ -34,10 +33,30 @@ class ExampleViewModel(application: Application) : AndroidViewModel(application)
     private val gatt = viewModelScope.keepAliveGatt(
         application,
         bluetoothAdapter.getRemoteDevice(MAC_ADDRESS),
-        disconnectTimeoutMillis = 5_000L // 5 seconds
+        disconnectTimeoutMillis = 5_000L, // 5 seconds
+        onEvent = { event ->
+            event.onConnected {
+               // Actions to perform on initial connect *and* subsequent reconnects:
+               discoverServicesOrThrow()
+            }
+            event.onDisconnected { disconnectInfo ->
+                // Actions to perform on disconnect, or after a failed connection attempt
+                if (disconnectInfo.wasConnected) { 
+                    /*
+                        True if this disconnection event follows a successful connection, false
+                        if it represents a failed connection attempt.
+                    */
+                } else if (disconnectInfo.connectionAttempt > 1) {
+                    /* 
+                        Value tracking the current connection attempt in the life span of the
+                        keepAliveGatt. Starts at 1 and increments by 1 for each connection attempt,
+                        whether successful or not.
```

Comment:
Won't this be "connection attempts since last successful connection"? (i.e. **not** life span of `keepAliveGatt`...whether successful or not)

Reply:
That was the original intention. I realized, though, that foregrounding and backgrounding the app again would also reset that behavior as well. As such I figured it might be cleaner to simply let the app manage the current index-offset for backoff purposes and have ABLE simply report the current connection attempt value.

Reply:
Spoke offline and was given clarification as to how this approach works well.
In summary: event handler can reset it's offset any time `wasConnected` is `true` (or any other condition that is desirable)